index.bs

```diff
@@ -349,8 +344,8 @@ spec: CSP; urlPrefix: https://www.w3.org/TR/CSP3/#
 
       1. Otherwise, [=set/for each=] <a>permissions-source-expression</a> |item| in the <a>allowlist</a>:
 
-        1. If the result of running <a>Does url match expression in origin with redirect count?</a>
-           on <var>origin</var>, |item|, <var>origin</var>, and 0 is true then return true.
+        1. If the result of running [=Does url match expression in origin with redirect count?=]
+           on a [=url=] of <var>origin</var>, |item|, <var>origin</var>, and 0 is true then return true.
```

Comment:
"url of origin" isn't a thing, unfortunately -- I think we really need to do something like

1. let |url| be the result of calling the [[=url parser=]](https://url.spec.whatwg.org/#concept-url-parser) on the [[=serialization of an origin|serialization=]](https://html.spec.whatwg.org/multipage/browsers.html#ascii-serialisation-of-an-origin) of |origin|
2. If the result of running ... on |url|, |item|, |origin| and 0...